### DISCUSSION TITLE
Sitemap - Hide specific content

### DISCUSSION THREAD

**Discussion by vladdes** [patrickdemooij9/SeoToolkit.Umbraco#171] [Q&A]
Hi, is it possible to hide specific content? I seem to only be able to set those on doc type level. Cheers!

Edit: I see now that I am supposed to get that option (I think) but getting some nice errors. 
 Possibly unhandled rejection: {"data":{"ExceptionMessage":"Unable to resolve service for type 'SeoToolkit.Umbraco.Common.Core.Collections.SeoDisplayCollection' while attempting to activate 'SeoToolkit.Umbraco.Common.Core.Controllers.SeoContentController'.","ExceptionType":"System.InvalidOperationException, System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e","StackTrace":"   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)\r\n   at lambda_method5069(Closure , IServiceProvider , Object[] )\r\n   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)"},"status":500,"config":{"method":"GET","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","url":"backoffice/SeoToolkit/SeoContent/Get?contentId=1258","headers":{"Accept":"application/json, text/plain, */*","X-Requested-With":"XMLHttpRequest","X-UMB-XSRF-TOKEN":"CfDJ8HInIqu2PMdOmNMhCpB421izOtJcjBKn2hax1wmMh9g1p3lWcQN6NtL5Gnhgj2J9LL2FU4ydru_JrLyHvPr_B1d-9ClOQA2qbkShPeDUj7Igcrod5HvWAAk5ZhOBNGJygSBZ-b2Vpa5PoGr_yvDl0yrWVjtvJKJYSpRjO3h738UMl0-xn4Y12XLCf8f7g_2rTA"}},"statusText":"","xhrStatus":"complete"}
 
 This using version 1.1 of seotoolkit on umbraco 9.2.2

**Comment by patrickdemooij9** [comment]
There currently isn't an interface for setting the sitemap options on content level though it is something that I want to add to the package.
As for the errors, do you only have the sitemap package installed or the whole SeoToolkit package?
